Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1486,12 +1515,12 @@ void CGContextSetStrokeColorWithColor(CGContextRef context, CGColorRef color) {
 }
 
 /**
- @Status Stub
- @Notes Since we are currently missing Color Space support, this will need to be implemented.
```

Comment:
actually patterns is using them.

---
In reply to: [92969413](https://github.com/Microsoft/WinObjC/pull/1586#discussion_r92969413) [](ancestors = 92969413)